tests/unit/test_sensors.py

```diff
@@ -0,0 +1,464 @@
+import json
+import os
+
+import numpy as np
+from pytest import approx
+
+from rocketpy.mathutils.vector_matrix import Matrix, Vector
+from rocketpy.tools import euler_to_quaternions
+
+# calisto standard simulation no wind solution index 200
+SOLUTION = [
+    3.338513236767685,
+    0.02856482783411794,
+    50.919436628139216,
+    1898.9056294848442,
+    0.021620542063162787,
+    30.468683793837055,
+    284.19140267225384,
+    -0.0076008223256743114,
+    0.0004430927976100488,
+    0.05330950836930627,
+    0.9985245671704497,
+    0.0026388673982115224,
+    0.00010697759229808481,
+    19.72526891699468,
+]
+UDOT = [
+    0.021620542063162787,
+    30.468683793837055,
+    284.19140267225384,
+    0.0009380154986373648,
+    1.4853035773069556,
+    4.377014845613867,
+    -9.848086239924413,
+    0.5257087555505318,
+    -0.0030529818895471124,
+    -0.07503444684343626,
+    0.028008532884449017,
+    -0.052789015849051935,
+    2.276425320359305,
+]
+
+
+def test_accelerometer_prints(noisy_rotated_accelerometer, quantized_accelerometer):
+    """Test the print methods of the Accelerometer class. Checks if all
+    attributes are printed correctly.
+    """
+    noisy_rotated_accelerometer.prints.all()
+    quantized_accelerometer.prints.all()
+    assert True
+
+
+def test_gyroscope_prints(noisy_rotated_gyroscope, quantized_gyroscope):
+    """Test the print methods of the Gyroscope class. Checks if all
+    attributes are printed correctly.
+    """
+    noisy_rotated_gyroscope.prints.all()
+    quantized_gyroscope.prints.all()
+    assert True
```

Comment:
Parametrize your tests cases to gain scalability and also help other developers reading this file in the future.

```suggestion
@pytest.mark.parametrize("gyroscope", ["noisy_rotated_gyroscope", "quantized_gyroscope"])
def test_gyroscope_prints(gyroscope):
    """Test the print methods of the Gyroscope class. Checks if all
    attributes are printed correctly.
    """
    gyroscope.prints.all()
    assert True
```

Reply:
addressed in https://github.com/RocketPy-Team/RocketPy/pull/583/commits/8c71432dc6e87d028bb774d7ea5dd67441a92adb